runtime/wasm/src/prepare.rs

```diff
@@ -138,31 +118,29 @@ impl<'a> ContractModule<'a> {
                 _ => continue,
             };
 
-            let Type::Function(ref _func_ty) = types
-                .get(*type_idx as usize)
-                .ok_or_else(|| Error::Instantiate)?;
+            let Type::Function(ref _func_ty) =
+                types.get(*type_idx as usize).ok_or_else(|| Error::Instantiate)?;
 
             // TODO: Function type check with Env
-			/*
-
-			let ext_func = env
-				.funcs
-				.get(import.field().as_bytes())
-				.ok_or_else(|| Error::Instantiate)?;
-			if !ext_func.func_type_matches(func_ty) {
-				return Err(Error::Instantiate);
-			}
-			*/
+            /*
```

Comment:
?

Reply:
Just reformatted the file, this was here before.